src/cms/templates/regions/region_form.html

```diff
@@ -154,11 +154,19 @@ <h3 class="heading font-bold text-white">{% trans 'Extended Settings' %}</h3>
 
                 <!-- Options related to page-instance-specific permissions -->
                 <span class="block p-2 mt-2 text-xl font-bold">{% trans 'Page permissions' %}</span>
-                <div class="p-2">
+                <div class="p-2 pb-4 border-b solid border-gray-400">
                     {% render_field form.page_permissions_enabled %}
                     <label for="{{ form.page_permissions_enabled.id_for_label }}" class="font-bold cursor-pointer p-2">{{ form.page_permissions_enabled.label }}</label>
                     <div class="my-2 text-s text-gray-600">{{ form.page_permissions_enabled.help_text }}</div>
                 </div>
+
+                <!-- Option for including/excluding administrative devision from region name -->
+                <span class="block p-2 mt-2 text-xl font-bold">{% trans 'Region name option' %}</span>
+                <div class="p-2">
+                    {% render_field form.administrative_division_included %}
+                    <label for="{{ form.administrative_division_included.id_for_label }}" class="font-bold cursor-pointer p-2">{{ form.administrative_division_included.label }}</label>
+                    <div class="my-2 text-s text-gray-600">{{ form.administrative_division_included.help_text }}</div>
+                </div>
```

Comment:
I'd propose to move this field beneath the `administrative_division` field...

Reply:
Yes, you are right, at this position it's more reasonable :+1: